aya/src/programs/mod.rs

```diff
@@ -410,6 +410,39 @@ impl Program {
             Self::CgroupDevice(p) => p.fd(),
         }
     }
+
+    /// Returns information about a loaded program with the [`ProgramInfo`] structure.
+    ///
+    /// This information is populated at load time by the kernel and can be used
+    /// to get kernel details for a given [`Program`].
+    pub fn program_info(&self) -> Result<ProgramInfo, ProgramError> {
```

Comment:
Why `program_info` and not `info`?

Reply:
Done! Also fixed up the existing API so that we're consistent.